app/main.py

```diff
@@ -1,3 +1,52 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+
+from app.car import Car
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as config:
+        config = json.load(config)
+        for customer in config["customers"]:
+            res = {}
+            print(f"{customer['name']} has {customer['money']} dollars")
+
+            for shop in config["shops"]:
+
+                car = Car(
+                    config["FUEL_PRICE"],
+                    customer["car"]["fuel_consumption"],
+                    customer["location"], shop["location"]
+                ).calculate_trip_cost()
+
+                sh = Shop(
```

Comment:
Please use more descriptive variable names. The current names 'car' and 'sh' do not adequately convey that they are related to prices. Consider renaming them to reflect their association with pricing